src/Control/Monad/Logic/Sequence.hs

```diff
@@ -30,11 +30,12 @@ module Control.Monad.Logic.Sequence
   , observe
   , observeMaybeT
   , observeMaybe
+  , chooseSeqT
```

Comment:
Is there a better name for this? The only reason we provide it as a "primitive" is so we can write it in a way that take advantage of the internals, but maybe `choose` (without the `SeqT`) is a better name?